### ISSUE TITLE
Remove dependencies

### ISSUE THREAD

**Issue by ChrisBAshton** [bbc/news-vj-sharetools#1]
ShareTools has a 'news bootstrap' dependency (in reality, this is just a jQuery dependency, aliased to `news.$`), and a 'template_engine' dependency. _We should make the sharetools self-contained and dependency free_.

For the record, we use John Resig's template engine:

``` javascript
/* jshint ignore:start */

// John Resig - http://ejohn.org/ - MIT Licensed
define(function () {
  var cache = {},
      tmpl = function tmpl(str, data) {
    // Figure out if we're getting a template, or if we need to
    // load the template - and be sure to cache the result.

    var fn = !/\W/.test(str) ?
      cache[str] = cache[str] ||
        tmpl(document.getElementById(str).innerHTML) :

      // Generate a reusable function that will serve as a template
      // generator (and which will be cached).
      new Function("obj",
        "var p=[],print=function(){p.push.apply(p,arguments);};" +

        // Introduce the data as local variables using with(){}
        "with(obj){p.push('" +

        // Convert the template into pure JavaScript
        str
          .replace(/[\r\t\n]/g, " ")
          .split("<%").join("\t")
          .replace(/((^|%>)[^\t]*)'/g, "$1\r")
          .replace(/\t=(.*?)%>/g, "',$1,'")
          .split("\t").join("');")
          .split("%>").join("p.push('")
          .split("\r").join("\\'")
      + "');}return p.join('');");

    // Provide some basic currying to the user
    return data ? fn( data ) : fn;
  };
  return tmpl;
});

/* jshint ignore:end */
```

**Comment by fewstera** [comment]
@ChrisBAshton this is a really good idea, we can host the template engine separately and bower it in.

Only issue that might come of this, is PAL name their jquery - `jquery-1` and `jquery-2`. And I think we just use `jquery` in the scaffold

**Comment by ChrisBAshton** [comment]
Started work in #2.

**Comment by ChrisBAshton** [comment]
Fixed in #2.
